test/Main.hs

```diff
@@ -12,5 +13,5 @@ main =
       -- FIXME: both `Integration.test` and `BotInterface.test`
       -- start own cluster to run tests, probably, need better solution in future
       [ Integration.test
-      , BotInterface.test
+      -- , BotInterface.test
```

Comment:
Was it temporary disabled by commenting out or something broken?

Reply:
Yeah, I forgot to uncomment it. Thanks for pointing it out.